src/02-data-types.js

```diff
@@ -79,7 +97,13 @@ function convertToWhisperText(text) {}
  *  checkIfCharacterIsInString("hello there", "a");
  *  //> false
  */
-function checkIfCharacterIsInString(text, character) {}
+function checkIfCharacterIsInString(text, character) {
+  if (text.includes(character)) {
```

Comment:
Remember that the return value of the .includes method is a boolean. We could refactor this function to just return the value of text.includes(character)